ludger14.html

```diff
@@ -0,0 +1,203 @@
+        height: 130px;
+        border: 1px solid #ccc;
+        padding-left: 8px;
+        margin-top: 10px;
+        font-size: 15px;
+        display: inline-block;
+        outline: 0;
+        resize: vertical;
+    }
+    .contato input[type=submit] {
+        cursor: pointer;
+        background-color: #249de3;
+        color: white;
+        border: 0;
+        width: 100px;
+        height: 30px;
+        display: block;
+        border-radius: 30px;
+        margin-top: 15px;
+        font-size: 15px;
+    }
+    section.pre-footer{
+        width: 100%;
+        padding: 20px 0;
+        background-color: gray;
+        text-align: center;
+    }
+    .pre-footer p{
+        font-size: 22px;
+        color: white;
+    }
+    footer{
+        width: 100%;
+        padding: 20px 0;
+        background-color: rgba(0, 0, 0, 0.9);
+        text-align: center;
+    }
+    footer p{
+        font-size: 22px;
+        color: white;
+    }
+</style>
+<body>
+    <header>
+        <div class="center">
+            <div class="logo">
+                <h1><a href="index.html">TOTI</a></h1>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">Sobre</a></li>
+                    <li><a href="#">Contato</a></li>
+                </ul>
+            </nav>
+            <div class="clear"></div>
+        </div>
+    </header>
+    <section class="banner"></section>
+    <section class="sobre">
+        <div class="center">
+            <h1>Quem Somos</h1>
+        </div>
+    </section>
+    <section class="text-sobre">
+        <div class="center">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut libero arcu, lacinia id blandit sed, bibendum quis est. Etiam eget ante dui. Morbi augue quam, ullamcorper non nisi ac, elementum porta elit. Vivamus ac metus porta, molestie mauris eu, elementum lacus. Nam vel nibh et mauris efficitur tempus. Maecenas felis orci, luctus vitae pulvinar commodo, aliquet eu enim. Mauris vel lacus eu nibh venenatis dignissim eu eu lorem. Aenean vehicula lacus nibh, eget fringilla orci tempus sit amet. Pellentesque venenatis diam ut nulla imperdiet mattis.</p>
+
+            <p>Cras porttitor lacus ut tortor mattis malesuada. Nam venenatis urna at ex volutpat bibendum. Mauris augue lacus, vestibulum eget tempus at, ultrices ut purus. Vestibulum auctor arcu at enim tincidunt, ac ultrices arcu dictum. Vestibulum lorem massa, scelerisque sit amet volutpat a, facilisis at urna. Nullam eu leo sem. Curabitur eros orci, pretium at diam non, ultrices condimentum lacus. Vivamus eleifend tortor ac molestie gravida. Cras ultricies sem sit amet nibh tristique commodo. Nulla sit amet sagittis mi, vitae iaculis felis. Aenean pharetra sit amet erat eget ornare.</p>
+        </div>
+    </section>
+    <section class="sobre">
+        <div class="center">
+            <h1>Entre em Contato</h1>
```

Comment:
Usar um nível de título mais baixo (trocar o h1)
```suggestion
            <h2>Entre em Contato</h2>
```